parser.py

```diff
@@ -0,0 +1,71 @@
+import graph
+import tokenizer
+# from abc import ABC, abstractmethod
```

Comment:
Code should should be removed instead of being commented out. The old code can be viewed in the git history